oap-ape/ape-native/src/reader.cc

```diff
@@ -375,6 +376,45 @@ void Reader::close() {
   // delete options;
 }
 
+void Reader::preBufferRowGroups() {
+  if (currentBufferedRowGroup >= currentRowGroup) {
+    return;
+  }
+
+  int maxBufferCount = 100;  // TODO
```

Comment:
For the 1st call, will this do harm to performance? Given a case only have 100 RGs in total, will this no impacts on performance. I am thinking whether we can configure this pre-cache or pre-fetch in a smaller stripe size, like just 1 RG ahead of time. Your thoughts?

Reply:
`PreBuffer` will load data asynchronously, so it has no blocking time here.

In fact, big `maxBufferCount` is not recommended. It may increase threads count of process and requires more memory to hold pre buffered data.

In the doc of under API, it says buffering only 1 row group may be helpful.

So, we can set this `maxBufferCount` to 1 too.